cloudhub-2/modules/ROOT/pages/ch2-architecture.adoc

```diff
@@ -71,7 +71,7 @@ You can scale replicas vertically by selecting one of the available vCore sizes:
 [%header,cols="4*a"]
 |===
 |vCore Size | Total Memory | Heap Memory | Storage
-|0.1        |1 GB          |500 MB       | 8 GB
+|0.1        |1.2 GB        |500 MB       | 8 GB
```

Comment:
shouldn't the heap memory goes up to 600?

Reply:
```suggestion
|0.1        |1.2 GB        |600 MB       | 8 GB
```